packages/boot/tools/supports.ts

```diff
@@ -457,13 +457,16 @@ export const makeSwingsetTestKit = async (
       },
     });
   }
+  const swingsetEnv = {
+    DEBUG: 'label-instances',
+  };
```

Comment:
Is this only for debugging, and still off in production? How?

If this is to be on in production, there are serious security issues mentioned all too briefly at https://github.com/Agoric/agoric-sdk/blob/62ea79368de79a697b8ef04ea9946cf621c81239/packages/swingset-liveslots/src/virtualObjectManager.js#L38-L41

So if this is to be on in production, who can see these labels? Given that they are on a public property, this includes: Why can see the objects carrying these labels?

Reply:
The `boot` package is only for testing.

Reply:
Resolving